bitnami/memcached/templates/statefulset.yaml

```diff
@@ -0,0 +1,138 @@
+{{- if eq .Values.architecture "high-availability" }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "memcached.fullname" . }}
+  labels: {{- include "memcached.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels: {{- include "memcached.matchLabels" . | nindent 6 }}
+  replicas: {{ .Values.replicaCount }}
+  serviceName: {{ template "memcached.fullname" . }}
```

Comment:
The current service is not a headless service, right?

Reply:
It is if statefulset